apps/api/src/app/import/import.service.ts

```diff
@@ -94,7 +97,8 @@ export class ImportService {
             value,
             assetProfile.currency,
             userCurrency
-          )
+          ),
+          isDuplicate: false
```

Comment:
Can you please move it below `isDraft` to keep the alphabetical order?

Reply:
Can you write: `isDuplicate: false // TODO: Use evaluated state`

In a follow up version, it should work the same way for dividend :)